src/reanimated2/animation/spring.ts

```diff
@@ -88,8 +79,8 @@ export const withSpring = ((
         return true;
       }
 
-      if (config.configIsInvalid) {
-        // We don't animate wrong config
+      if (config.skipAnimation) {
+        // If we use duration we want to wait the provided time before stopping
         if (config.useDuration) return false;
```

Comment:
There's no curly braces after `if`, can we add them and also check why linter doesn't complain about it? We should have `curly` rule enabled. Also, what's the conclusion when it comes to the expected behavior when config is invalid? Why does it depend on whether we use duration or not?
```suggestion
        if (config.useDuration) {
          return false;
        }
```

Reply:
I've thought we have decided that changing this behaviour should be done in separate PR (to avoid big changes on a reviewed PR), but since this is not a big change I'll delete it. It is because of the previous behaviour (we used to wait duration time before updating the value).

Reply:
The rule 'curly' is only enabled in plugin

Reply:
We can enable it (we have vialotion of this rule only in one file)

Reply:
Thanks for checking this, let's do it (enable it everywhere)